doc/tutorials/ostro-oic-tutorial/step4/http-light-server.fbp

```diff
@@ -0,0 +1,29 @@
+# This file is part of the Soletta Project
+#
+# Copyright (C) 2015 Intel Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+INPORT=power.IN:POWER
+INPORT=state.IN:STATE
+
+OUTPORT=power.OUT:POWER
+OUTPORT=state.OUT:STATE
+
+power(http-server/int:path="/power",port=8002)
+state(http-server/boolean:path="/state",port=8002)
+
+power OUT -> IN print_power(console)
+state OUT -> IN print_state(console)
+
```

Comment:
On 10-05-2016 09:42, Flávio Vinicius Alvares Ceolin wrote:
> In doc/tutorials/ostro-oic-tutorial/step4/http-light-server.fbp
> <https://github.com/solettaproject/soletta/pull/2000#discussion_r62663711>:
>
>> +# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
>> +# See the License for the specific language governing permissions and
>> +# limitations under the License.
>> +
>> +INPORT=power.IN:POWER
>> +INPORT=state.IN:STATE
>> +
>> +OUTPORT=power.OUT:POWER
>> +OUTPORT=state.OUT:STATE
>> +
>> +power(http-server/int:path="/power",port=8002)
>> +state(http-server/boolean:path="/state",port=8002)
>> +
>> +power OUT -> IN print_power(console)
>> +state OUT -> IN print_state(console)
>> +
>
> could be only one |console| node

But this way it prints which information is being printed.

> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/solettaproject/soletta/pull/2000/files/e958bb17858fee016474856da50253af778bee1d#r62663711>
>